docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| napari term        | Description                                                                                                                                                                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
+| hook specification | A function signature that defines the API that a plugin developer must adhere to when writing the function that we promise to call somewhere in the napari codebase.                                                                                                                                                                        |
```

Comment:
* Possibly useful links:  https://napari.org/docs/dev/plugins/hook_specifications.html
* "somewhere in the napari codebase" is much too vague!
* Honestly, hook specs are more about **when** an action will be triggered, not **where**. I'd focus on that in this explanation.
* If it's difficult to explain without a concrete example, I'd add a concrete example here.